README.md

```diff
@@ -236,6 +238,28 @@ without the agent. In order to do that, the `provisionVMAgent` flag must be set
 provisioning time, via whichever API is being used. We will provide more details on
 this on our wiki when it is generally available. 
 
+#### __Extensions.GoalStatePeriod__
```

Comment:
I'm not sure if we should expose this in the conf mainly because small things like the VM status is dependent on this.

Reply:
There is a legitimate need to change the default. I added a note in the documentation. Also, changes are logged locally and to telemetry.